lightgbm_ray/main.py

```diff
@@ -750,7 +754,7 @@ def handle_actor_failure(actor_id):
             else:
                 logger.debug("Couldn't obtain unique addresses, trying again.")
     if machines:
-        logger.debug(f"Obtained unique addresses in {n} attempts.")
+        logger.debug(f"Obtained unique addresses in {attempts} attempts.")
```

Comment:
This should reflect the actual number if iterations it took to obtain unique addresses!